Assets/02.Scripts/02-8. UI/DailyReport/DailyReportPresenter.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using UnityEngine;
+
+public class DailyReportPresenter : MonoBehaviour
+{
+    [SerializeField]
+    private DailyReportModel _dailyReportModel;
+    
+    private UI_DailyReport _dailyReport;
+
+    private void Awake()
+    {
+        _dailyReport = GetComponent<UI_DailyReport>();
+    }
+
+    private void Start()
+    {
+        _dailyReportModel.OnDailyReportChanged += OnChangeDailyReportData;
+    }
+
+    private void OnChangeDailyReportData(DailyReportData data)
+    {
+        _dailyReport.RefreshDailyReport(data);
+    }
```

Comment:
```suggestion
    private void Start()
    {
        _dailyReportModel.OnDailyReportChanged += _dailyReport.RefreshDailyReport;
    }

```
굳이 View의 메서드 호출부를 별도 메서드로 빼줄 필요는 없을 것 같습니다. 메서드명을 보니 딱히 가독성에 도움이 되는 것 같지도 않아서요